src/app/signin/page.tsx

```diff
@@ -0,0 +1,162 @@
+"use client"
+
+import Image from "next/image";
+import Link from "next/link";
+import "./page.css";
+import { useRecoilState } from "recoil";
+import { authState, AuthState} from "../recoil/atoms"
+
+function SignIn() {
+  const [auth, setAuth] = useRecoilState<AuthState>(authState)
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>)=> {
+    const { name, value} = e.target;
+    setAuth ((prevAuth) => ({
+      ...prevAuth,
+      [name]: value,
+      errors: {...prevAuth.errors, [name]: ""}
+    }))
+  }
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault()
+    const {email, password} = auth;
+    let formIsValid = true;
+
+    const newErrors = {email: "", password: ""};
+
+    if (email.trim() === "") {
+      formIsValid = false;
+      newErrors.email = "Email is required"
+    }
+
+     if (password.trim() === "") {
+      formIsValid = false;
+      newErrors.password = "Password is required"
+    }
+
+    if (!formIsValid) {
+     setAuth ((prevAuth) => ({
+      ...prevAuth,
+      errors: newErrors,
+
+      }));
+    } else {
+      // Handle successfull login
+      console.log("Logged in:", email)
+    }
+    
+  };
+
+  return (
+    <section className="login-bg">
+    <div className="login-container">
+      <div className="login-header">
+      <Link
+        href="/"
+        style={{ 
+          textDecoration: 'none'
+        }}>
+        <div className="logo-container">
+          <Image src="/icons/logo.png" alt="logo" width={60} height={60} />
+          <p>Song Sieve </p>
+        </div>
+      </Link>
+      </div>
+
+      <div className="form-container">
+        <h2>Sign In</h2>
+        <form onSubmit={handleSubmit}>
+          <div className="form-group">
+            <input
+              type="email"
+              name="email"
+              className="form-control"
+              placeholder="Email"
+              value={auth.email}
+              onChange={handleInputChange}
+            />
+            <Image
+              src="/icons/email-icon.svg"
+              alt="Email"
+              className="input-icon"
+              width={25}
+              height={25}
+            />
+            <div className="error">{auth.errors.email}</div>
+          </div>
+
+          <div className="form-group">
+            <input
+              type="password"
+              name="password"
+              className="form-control"
+              placeholder="Password"
+              value={auth.password}
+              onChange={handleInputChange}
+            />
+            <Image
+              src="/icons/lock-icon.svg"
+              alt="Password"
+              className="input-icon"
+              width={25}
+              height={25}
+            />
+            <div className="error">{auth.errors.password}</div>
+          </div>
+
+          <div className="Forget-password">
```

Comment:
For the future, it's better practise to use lowercase 1st letter for naming classes - "forget-password". 
(But I think it was an accident:)